src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/RequestPayloadValueResolver.php

```diff
@@ -161,7 +160,7 @@ private function mapQueryString(Request $request, string $type, MapQueryString $
             return null;
         }
 
-        return $this->serializer->denormalize($data, $type, null, $attribute->serializationContext + self::CONTEXT_DENORMALIZE);
+        return $this->serializer->denormalize($data, $type, CsvEncoder::FORMAT, $attribute->serializationContext + self::CONTEXT_DENORMALIZE);
```

Comment:
Maybe we can inline the value to save loading an extra class?

Reply:
5d1014d9d56e9c43a71e13c1d65122edcdfa5438